### DISCUSSION TITLE
Error with Outline + Authentik OIDC

### DISCUSSION THREAD

**Discussion by mlcsthor** [outline/outline#5988] [Self Hosting]
Hi, I have a problem with the configuration of Outline using Authentik as OIDC Provider. 
I'm redirected to the Authentik portal but after returning to Outline I get a code 502 with the following error in the logs:
```
{
	"error":"Operation timeout",
	"level":"error",
	"message":"Error during authentication",
	"stack":"UnauthorizedError: Operation timeout 
		at InvalidAuthenticationError (/opt/outline/build/server/errors.js:46:34)
		at accountProvisioner (/opt/outline/build/server/commands/accountProvisioner.js:59:54) 
		at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
}
```

**Comment by tommoor** [comment]
Outline cannot connect to Authentik – this is usually a DNS issue, so have a search in the discussions as it's come up many times.

**Reply by mlcsthor** [reply]
DNS was my first clue to solving the problem. But the same error also occurs if I use Authentik's IP instead of the domain name and I can ping Authentik domain from Outline container.
In addition, I can see the token request in Authentik's logs coming from Outline with a status 200
